torch_ipex/csrc/cpu/ShadeDataContext.h

```diff
@@ -116,7 +113,6 @@ struct ShadeDataContext {
         // All these tensors share same buffer of Tensor A with different storge offsets and elements.
         // So the context modification will impact all these tensors.
         if (check_tensor_own_whole_storage(tensor)) {
-          TORCH_INTERNAL_ASSERT_DEBUG_ONLY(shade_data_context->dil_tensor->get_size() == tensor.storage().capacity());
```

Comment:
Did you hit the assert? Or is it a reduandant assert?

Reply:
ditto